README.md

```diff
@@ -77,6 +77,37 @@ These workflows were built based on existing tools below:
 
     **NOTE:** File names can be wrong if run by Dockstore.
 
+### chksum_xam_to_interleaved_fq.cwl
+
+#### Inputs
+
+* `xam_in` - A **list** of two paired gzipped FastQ files.
```

Comment:
Not two paired gzipped fastq. Isn't this a [B|Cr]am file or list of?

Reply:
You're right!